src/transaction/transaction.rs

```diff
@@ -804,10 +895,17 @@ impl TransactionOptions {
             }
         }
     }
+
+    pub fn stop_heart_beat(mut self) -> TransactionOptions {
```

Comment:
Oh! It's the option.  I wonder it was the txn. I suggest `no_auto_heartbeat()` for consistency.